# Move Servo msg, include three param: servoid, targetPos, delay
Header header

uint8 servoId
uint16 targetPos
uint8 delay
